### ISSUE TITLE
Pushback via EFB going in wrong direction upon first click

### ISSUE THREAD

**Issue by Venorcis** [flybywiresim/a32nx#7246]
### Aircraft Version

Development

### Build info

```json
{
    "built": "2022-05-25T01:03:21+00:00",
    "ref": "refs/heads/master",
    "sha": "bba5b4acc558c277c021082e749a20001aecee4b",
    "actor": "2hwk",
    "event_name": "manual",
    "pretty_release_name": "master:bba5b4ac"
}
```

### Describe the bug

The tug goes into the opposite direction when clicking a certain direction on the EFB. Upon the second click it will correct itself. Further clicks seem to mess it up as well (might be alternating or something, not sure).

### Expected behavior

Tug should turn in the expected direction upon the first click.

### Steps to reproduce

1. Start pushback
2. Push left or right arrow once, observe wrong turning direction.
3. Push same error a second time, observe correction into the right direction.

### References (optional)

https://www.twitch.tv/videos/1310637697?t=00h07m40s

**Comment by 2hwk** [comment]
Note: this issue will be invalidated once flyPad OS 3 is merged into development as that has an entirely new pushback logic.
